packages/psss-server/src/connections/EntityConnection.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { PSSS_ENTITY, PSSS_HIERARCHY } from '../constants';
+import { ApiConnection } from './ApiConnection';
+
+const { ENTITY_URL = 'http://localhost:8050/v1' } = process.env;
```

Comment:
```suggestion
const { ENTITY_SERVER_API_URL = 'http://localhost:8050/v1' } = process.env;
```
I think that's the convention we have for naming env vars for the server urls

Reply:
Good point (it seems that it's actually `ENTITY_API_URL`)